go.mod

```diff
@@ -8,9 +8,9 @@ require (
 	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/ahmetb/gen-crd-api-reference-docs v0.3.0
 	github.com/coreos/go-systemd/v22 v22.5.0
-	github.com/gardener/etcd-druid v0.19.2
-	github.com/gardener/gardener v1.81.6
-	github.com/gardener/machine-controller-manager v0.48.1
+	github.com/gardener/etcd-druid v0.20.1
+	github.com/gardener/gardener v1.83.2
```

Comment:
Is it possible to update to v1.83.3? In v1.83.3 an an issue in the extension library that prevents CPM for hibernated Shoots is resolved.

Reply:
Done.